tests/test_detect_repo_type.py

```
@@ -499,6 +456,99 @@ def test_services_dir_scores_monorepo(self, tmp_path):
+        assert "(confidence:" in first_line
+
+    def test_evidence_lines_indented(self, tmp_path):
+        """Evidence lines start with '  - '."""
+        (tmp_path / "packages").mkdir()
+        result = subprocess.run(
+            [sys.executable, str(self._script), str(tmp_path)],
+            capture_output=True, text=True,
+        )
+        assert result.returncode == 0
+        evidence_lines = [
+            l for l in result.stdout.splitlines()[1:]
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)